src/content/guides/index.md

```diff
@@ -7,6 +7,6 @@ contributors:
 
 This section contains guides for understanding and mastering the wide variety of tools and features that webpack offers. The first is a simple guide that takes you through [installation](/guides/installation).
 
-The guides get more and more advanced as you go on. Most serve as a starting point, and once completed you should feel more comfortable diving into the actual [documentation](/configuration).
+The guides get more and more advanced as you go on. Most serve as a starting point and once completed you should feel more comfortable diving into the actual [documentation](/configuration).
 
-W> The output shown from running webpack in the guides may differ slightly from the output of newer versions. This is to be expected. As long as the bundles look similar and run correctly, then there shouldn't be any issues. If you do come across an example that seems to be broken by a new version, please create an issue and we will do our best to resolve the discrepancy.
+W> The output shown from running webpack in the guides may differ slightly from the output of newer versions. This is to be expected. As long as the bundles look similar and run correctly, then there shouldn't be any issues. If you come across an example that seems to be broken by a new version, please create an issue and we will do our best to resolve the discrepancy.
```

Comment:
again, this is unwanted!
Use of `do` makes sense completely.

Reply:
@hemal7735 Please reduce the tone of your comments.

Reply:
I apologize if it felt that way @bhavya9107 and @montogeek . 😞

Reply:
I must admit this grammar improvement isn't shining :(

Reply:
Documents are meant to be read. According to standard technical writing guidelines, readability is an important aspect. As `do` activity is implicit with verbs, so removing `do` makes it more readable.

Reply:
While “do” here isn’t absolutely necessary, it does provide emphasis and sets the sentence in contrast to the one preceding it. I’d recommend keeping “do” here.